addon/components/lt-body.js

```diff
@@ -350,6 +365,25 @@ export default Component.extend({
     }
   },
 
+  _onFocusedRowChanged: Ember.observer('table.focusIndex', function() {
+    run.schedule('afterRender', null, () => {
+      let row = this.$('tr.has-focus');
```

Comment:
I am unable to make it work with scrollToRow. I will leave it like this until someone figure this out.